src/store/reducers/index.js

```diff
@@ -0,0 +1,7 @@
+import { combineReducers} from 'redux';
+import lists from './list';
+import addTodoInList from './add_todo_in_list';
+export default combineReducers({
+    lists,
+    addTodoInList
```

Comment:
use proper naming convention
like
export default combineReducers({
    toDoItemsList
)}

other developers should know the purpose of variable by just looking at it